src/rs.cpp

```diff
@@ -69,17 +70,17 @@ struct rs2_processing_block_list
 
 struct rs2_sensor : public rs2_options
 {
-    rs2_sensor(rs2_device parent,
-        librealsense::sensor_interface* sensor) :
-        rs2_options((librealsense::options_interface*)sensor),
-        parent(parent), sensor(sensor)
+    rs2_sensor( rs2_device parent,
+        librealsense::sensor_interface * sensor ) :
+        rs2_options( ( librealsense::options_interface * )sensor ),
+        parent( parent ), sensor( sensor )
     {}
 
     rs2_device parent;
-    librealsense::sensor_interface* sensor;
+    librealsense::sensor_interface * sensor;
 
-    rs2_sensor& operator=(const rs2_sensor&) = delete;
-    rs2_sensor(const rs2_sensor&) = delete;
+    rs2_sensor & operator=( const rs2_sensor & ) = delete;
```

Comment:
Now I see you reformatted all of rs.cpp :)
Can we undo this please?

Reply:
done